scripts/irods/test/test_ireg.py

```diff
@@ -209,3 +209,11 @@ def test_ireg_double_slashes__issue_3658(self):
         self.admin.assert_icommand('iscan {0}'.format(os.path.abspath(dirname)))
         shutil.rmtree(os.path.abspath(dirname), ignore_errors=True)
 
+    def test_ireg_file_with_unresolvable_owner__issue_4040(self):
+        filename = '../test/unresolvable_uid_testfile__issue_4040'
+        fullpath = os.path.abspath(filename)
+        # if file is there (should be for a package installation)
+        # attempt the ireg, should pass now that 4040 is fixed
+        if os.path.isfile(fullpath):
```

Comment:
Should the test fail in the event that the file does not exist or otherwise indicate that the test did not run?

Reply:
thought about issuing a pass or skipIf, but just not running 'ireg' if the file wasn't there seemed easier.   how would we indicate that it didn't run?